release 3.5.3 (#3770)
Codeception/CodeceptJS: CHANGELOG.md

## 3.5.3

🛩️ Features

> * [Playwright]: providing more steps to check the network traffics #3748 - by @ngraf @KobeNguyenT

Reviewer: ```suggestion
* [Playwright] Provide new steps to check network traffic #3748 - by @ngraf @KobeNguyenT
```

```js
// recording traffics and verify the traffic
  await I.startRecordingTraffic();
  I.amOnPage('https://codecept.io/');
  await I.seeTraffic({ name: 'traffics', url: 'https://codecept.io/img/companies/BC_LogoScreen_C.jpg' });
```

```js
// block the traffic
  I.blockTraffic('https://reqres.in/api/comments/*');
  await I.amOnPage('/form/fetch_call');
  await I.startRecordingTraffic();
  await I.click('GET COMMENTS');
  await I.see('Can not load data!');
```

```js
// check the traffic with advanced params
  I.amOnPage('https://openai.com/blog/chatgpt');
  await I.startRecordingTraffic();
  await I.seeTraffic({
    name: 'sentry event',
    url: 'https://images.openai.com/blob/cf717bdb-0c8c-428a-b82b-3c3add87a600',
    parameters: {
      width: '1919',
      height: '1138',
    },
  });
```

🐛 Bugfix

> * [retryStepPlugin] fix the retry step doesn't work when using global retry #3768 - by @KobeNguyenT

Reviewer: ```suggestion
* [retryStepPlugin] Fix retry step when using global retry #3768 - by @KobeNguyenT
```
